### DISCUSSION TITLE
Define what is called "ordinary user"

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#419] [General]
I hate having to re-download the software installation every time after installing the system. Every time I have to wait for the download to end, and then I have to wait for the installation. I have to go through a long wait, which is very annoying.

What about the sideloading of the menu list I will only consider it an app store, because only the app store will put "specific things", and can only be downloaded from it.

So I look forward to the day when Appimage can be executed in helloSystem.

**Comment by louies0623** [comment]
No one user wants to use a system with a high learning threshold for work, I am very grateful In the past Microsoft and Apple were willing to lower the threshold for learning computers, so that users would not have to rely on terminals to install personal software.

**Reply by probonopd** [reply]
Yes, this is what we are aiming for. "Normal users" should never need the Terminal.

**Comment by louies0623** [comment]
Then let's define what is called "ordinary user", What is required to become a basic user, and what occupation they use, such as photographer, doctor ,  elderly care volunteers, etc, what equipment do they use, whether it is a laptop or a desktop or a POS, what software do they use to work, and whether they have a learning curve to become an advanced user, for example: the graphical interface for advanced settings.

**Reply by louies0623** [reply]
I will come first, will use the touch screen and calculator, the coffee shop staff, POS, may use Floreant to work, and sometimes he will adjust the price of coffee.

**Reply by probonopd** [reply]
helloSystem is definitely __not__ developed for touch.

It is a desktop operating system for personal computers operated by a mouse.

**Reply by louies0623** [reply]
Some creators will use this screen to draw, such as animators, cartoonists, illustrators, etc. We all need this function.
[SPA Animation Studio](https://youtu.be/EASbvJNQz0U)

![image](https://user-images.githubusercontent.com/44593430/209602743-9e76c6dd-4dda-4966-9458-290204cbd42f.png)
![image](https://user-images.githubusercontent.com/44593430/209602810-c98e4fc6-9816-487e-9564-59418dcc4562.png)

**Comment by probonopd** [comment]
A "normal user", a.k.a. "mere mortal", in this context is someone who has no technical background but wants to use a personal (desktop) computer to browse the internet, write letters, layout documents, edit photos, etc.

Someone who has been using Macs in the 80s and 90s for productivity work.

Someone who is not comfortable using the command line.

**Comment by louies0623** [comment]
So are one-handed users, or deaf, or blind, and people with SMA, classified as what you call normal users?

**Reply by probonopd** [reply]
I'd say those are users with special requirements.
